genienlp/ned/bootleg.py

```diff
@@ -30,80 +30,44 @@
         self.model_dir = f'{self.args.database_dir}/{self.args.bootleg_model}'
         self.config_path = f'{self.model_dir}/bootleg_config.yaml'
         self.cand_map = f'{self.args.database_dir}/wiki_entity_data/entity_mappings/alias2qids.json'
 
         self.entity_dir = f'{self.args.database_dir}/wiki_entity_data'
         self.embed_dir = f'{self.args.database_dir}/wiki_entity_data'
 
-        self.ned_domains = args.ned_domains
-        self.bootleg_post_process_types = args.bootleg_post_process_types
-
         with open(f'{self.args.database_dir}/wiki_entity_data/type_mappings/wiki/qid2typenames.json') as fin:
             self.entityqid2typenames = ujson.load(fin)
-        with open(f'{self.args.database_dir}/wiki_entity_data/type_mappings/wiki/type_vocab_to_wikidataqid.json') as fin:
-            self.type_vocab_to_typeqid = ujson.load(fin)
-            self.typeqid_to_type_vocab = {v: k for k, v in self.type_vocab_to_typeqid.items()}
-        with open(f'{self.args.database_dir}/es_material/typeqid2id.json') as fin:
-            self.typeqid2id = ujson.load(fin)
-        self.id2typeqid = {v: k for k, v in self.typeqid2id.items()}
-
-        ##### get mapping between wiki types and normalized almond property names
-        # keys are normalized types for each thingtalk property, values are a list of wiki types
-        self.almond_type_mapping = dict()
-
-        # a list of tuples: each pair includes a wiki type and their normalized type
-        self.wiki2normalized_type = list()
-
-        if self.args.almond_type_mapping_path:
-            # read mapping from user-provided file
-            with open(os.path.join(self.args.root, self.args.almond_type_mapping_path)) as fin:
-                self.almond_type_mapping = ujson.load(fin)
-            self.update_wiki2normalized_type()
-        else:
-            # this file contains mapping between normalized types and wiki types *per domain*
-            # we will choose the subset of domains we want via ned_domains
-            with open(
-                os.path.join(os.path.dirname(os.path.abspath(__file__)), 'database_files/almond_type_mapping.json')
-            ) as fin:
-                almond_type_mapping_all_domains = ujson.load(fin)
-            # only keep subset for provided domains
-            for domain in self.ned_domains:
-                self.almond_type_mapping.update(almond_type_mapping_all_domains[domain])
-            self.update_wiki2normalized_type()
-        #####
-
-        self.cur_entity_embed_size = 0
-
-        # Mapping between model directory and checkpoint name
-        model2checkpoint = {'bootleg_uncased_mini': 'bootleg_wiki.pth', 'bootleg_uncased_super_mini': 'bootleg_wiki.pth'}
-
-        self.ckpt_name, extension = model2checkpoint[self.args.bootleg_model].split('.')
-        self.model_ckpt_path = os.path.join(self.model_dir, self.ckpt_name + '.' + extension)
+        ###
+
+        self.ckpt_name = 'bootleg_wiki'
```

Comment:
This is a cmdline argument right?

Reply:
Checkpoint name used to be different for different bootleg models. Now it's all the same.